build.sh

```diff
@@ -149,5 +150,39 @@ configure_user(){
     echo '%wheel ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers.d/wheel-nopasswd
 }
 
+write_repo() {
+    # Save in-container artifacts about the container for archiving in Koji.
+    mkdir -p /build
+    git_state="clean";
+    if [[ $(git diff --shortstat 2> /dev/null | tail -n1) != '' ]]; then
```

Comment:
It seems like upstream uses the return code to make this determination. Is there a good reason to use text output comparison instead? 

https://github.com/git/git/blob/041fe8fc83770f95b09db4aa9d9b3783789eab08/contrib/completion/git-prompt.sh#L483-L484

Reply:
I've switched over to the upstream; much cleaner.